push-messaging-and-notifications/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Push Demo",
+  "short_name": "Push Demo",
+  "icons": [{
+        "src": "images/icon-192x192.png",
+        "sizes": "192x192"
+      }],
+  "start_url": "/index.html?homescreen=1",
+  "display": "browser",
```

Comment:
I think we're pushing (hah!) `"display": "standalone"` in the related blog posts, so perhaps go with that here, too?